l10n_fi_reconcile/models/account_bank_statement.py

```diff
@@ -109,7 +109,21 @@ def _get_auto_reconcile_new_aml_dicts(self, match_recs):
         auto reconciling complex situations, e.g. cash discounts.
         This stub method may be extended to handle those situations.
         """
-        return None
+        return []
+
+    def is_valid_finnish_match(self, match_recs, new_aml_dicts):
+        """
+        Check if this is a valid match and can be reconciled.
+        This method is intended to be extended by other modules to add domain specific checks.
```

Comment:
If this method should be extended, we should have a comprehensive docstring describing the purpose (more detail than `Check if this valid and can be reconciled`). The name indicates this has something to do with finnish matching: should extenders keep this in mind when extending? Also please describe the arguments in as much detail as you can.